src/Services/Search/ParametersHandler.php

```diff
@@ -65,24 +65,44 @@ public function handlePaginationAndSorting($search, $request)
         $config = $this->getConfig();
 
         $isSearch = strpos($request->getUri(), '/search') !== false;
+        $isFiltersSet = strpos($request->getUri(), '?attrib') !== false;
 
         $defaultSort = $isSearch ? $config->sorting->defaultSortingSearch : $config->sorting->defaultSorting;
 
         if (!in_array($defaultSort, Plugin::API_SORT_ORDER_AVAILABLE_OPTIONS)) {
             $defaultSort = 'item.score';
         }
 
-        $search->setSortingOptions($this->getSortingOptions($isSearch), $defaultSort);
+        $search->setSortingOptions($this->getSortingOptions($isSearch, $isFiltersSet), $defaultSort);
         $search->setItemsPerPage($this->getItemsPerPage($search), $this->getCurrentItemsPerPage($request, $search));
 
         return $search;
     }
 
-    /**
-     * @param boolean $isSearch
-     * @return array
-     */
-    public function getSortingOptions($isSearch)
+    public function getSortingOptions(bool $isSearch, bool $isFiltersSet): array
+    {
+        if (!$isSearch && !$isFiltersSet) {
+            return $this->getPlentymarketsSortingOptions();
+        }
+
+        return $this->getFindologicSortingOptions($isSearch);
+    }
+
+    public function getPlentymarketsSortingOptions(): array
+    {
+        /** @var CeresConfig $config */
+        $config = $this->getConfig();
+
+        $returnArray = [];
```

Comment:
`$returnArray` is not self explanatory, try to find a better name that suites the value of the variable.